capture-sdk/sdk/src/main/java/net/gini/android/capture/analysis/AnalysisHintsAnimator.java

```diff
@@ -19,10 +19,7 @@
 import androidx.core.view.ViewPropertyAnimatorListenerAdapter;
 
 /**
- * Created by Alpar Szotyori on 09.05.2019.
```

Comment:
[Clarification] Did you agree with Alpar to remove copyright and keep only `Internal use only`?